content/md/products-super-power/improve-data-quality.md

```diff
@@ -81,26 +105,26 @@ From the `data quality panel`, you can view attribute labels and options with sp
 ![Image data quality spellcheck on attribute option](../img/data-quality-spelling-option.png)
 
 
-# Improve data quality on variant products
+## Improve data quality on variant products
+
+If you work with variant products, you will see slight differences between simple products, product models and variant products.
 
-If you work with variant products, you will see slight differences between grades and recommendations for simple products, product models and variant products.
+To help you craft compelling product experiences, the **quality score** aims at helping you improve the data quality of your end-customer facing product data. The product your customer is interested in is the product variant. Therefore, the **quality score** is calculated and displayed only at the variant level.
 
 For **product models**:
 ![Image data quality product model panel](../img/data-quality-model-panel.png)
-- Data quality grades are calculated on the attributes that can be improved at the product model level.
 - Recommendations are given for attributes at the product model level.
 
 For **sub-product models**:
 ![Image data quality sub product model panel](../img/data-quality-sub-model-panel.png)
-- Data quality grades are calculated based on the attributes that can be improved at the product model and sub-product model levels
 - Recommendations are given for attributes at the product model and sub-product model levels. By clicking on an attribute name, you can go and improve an attribute in the product model or sub-product model.
 
 For **variant products**:
 ![Image data quality variant panel](../img/data-quality-variant-panel.png)
-- Data quality grades are calculated on the attributes that can be improved at all levels hence reflecting the data quality of the product that will be published. Data quality grades are therefore displayed in the left side bar only for product variant.
+- The quality score is calculated on the attributes that can be improved at all levels hence reflecting the data quality of the product that will be published.
 - Recommendations are given for all attributes by clicking on an attribute name, you can go and improve an attribute in the product model, the sub-product model or the variant product.
 
-# Improve data quality on attribute labels and Options
+## Improve data quality on attribute labels and options
 
 From the `Settings` you can check and improve the data quality of your attribute labels and options.
```

Comment:
From `Settings` / `Attributes`, you can check and improve the data quality of your attribute labels and options.